bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/widgets/Shell.java

```diff
@@ -125,7 +125,20 @@ public class Shell extends Decorations {
 	ToolTip [] toolTips;
 	long hwndMDIClient, lpstrTip, toolTipHandle, balloonTipHandle, menuItemToolTipHandle;
 	int minWidth = SWT.DEFAULT, minHeight = SWT.DEFAULT, maxWidth = SWT.DEFAULT, maxHeight = SWT.DEFAULT;
-	private int nativeZoom;
+	/**
+	 * the native zoom of the monitor the shell is drawn on
+	 * (Warning: This field is platform dependent)
+	 * <p>
+	 * <b>IMPORTANT:</b> This field is <em>not</em> part of the SWT
+	 * public API. It is marked public only so that it can be shared
+	 * within the packages provided by SWT. It is not available on all
+	 * platforms and should never be accessed from application code.
+	 * </p>
+	 *
+	 * @noreference This field is not intended to be referenced by clients.
+	 * @since 3.126
+	 */
+	public int nativeZoom;
```

Comment:
Don't do this, this allows writing into the field.
If you really need to share across packages then elevate the visibility of `getNativeZoom()` and add the `@noreference` to it so it's clear that it is only intended for internal usage

```suggestion
	private int nativeZoom;
```

Reply:
Introducing a public getter/setter will change the API and it is a Win32 specific implementation so we should not do that.